src/components/Basic/Dropdown/DropdownOption.tsx

```diff
@@ -46,10 +47,20 @@ export const DropdownOption: ParentComponent<{ option: ParentProps<OptionTokenPr
         return classes.join(' ');
     }
 
+    const { navigationActions } = useBaseComponent(props);
+
     return <div
+        ref={element}
+        //@ts-ignore
+        attr:disabled={props.option.disabled || undefined}
         onclick={() => onClickOption(props.option)}
+        onMouseOver={(e: MouseEvent) => (e.currentTarget as HTMLElement).focus()}
         class={optionClasses(props.option)}
-        style={props.option.style}
+        style={{...props.option.style, color: "initial"}}

```

Comment:
This is causing the following warning: `Unable to parse declaration: color - initial`. Can we remove it?